libraries/c_sdk/standard/ble/src/services/mqtt_ble/iot_ble_mqtt_transport.c

```diff
@@ -844,101 +955,103 @@ static MQTTStatus_t handleIncomingPingresp( MQTTPacketInfo_t * packet,
  * @param[in] buf A pointer to a buffer containing data to be sent out.
```

Comment:
The description says prototype. It may not be prototype anymore.

Reply:
Fixed